.github/workflows/build-and-publish.yml

```diff
@@ -0,0 +1,189 @@
+name: Build and Publish NuGet Package for all supported platforms
+
+on:
+  push:
+  pull_request:
+    branches:
+    - master
+
+env:
+  MAJOR: 4
+  MINOR: 1
+  RELEASE: 0
+  RUN: ${{ github.run_number }}
+
+jobs:
+  get-version:
+    runs-on: ubuntu-latest
+    outputs:
+      BUILD_ID: ${{ steps.get_build_id.outputs.BUILD_ID}}
+      APP_VERSION: ${{ steps.get_app_version.outputs.APP_VERSION}}
+    steps:
+    - name: Get New Build Number
+      id: get_build_id
+      shell: bash
+      run: |
+
+        # Get the build ID
+        if [[ "${{ github.event_name }}" == 'push' && "${{ github.ref_name }}" == "${{ github.event.repository.default_branch }}" ]]; then
+          # Fetch the latest version from the organization NuGet package
+          response=$(curl -s -L \
+            -H "Accept: application/vnd.github+json" \
+            -H "Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" \
+            -H "X-GitHub-Api-Version: 2022-11-28" \
+            https://api.github.com/orgs/Open-Systems-Pharmacology/packages/nuget/OSPSuite.SimModel.Solver.CVODES/versions)
+
+          # Log the raw response for debugging
+          echo "API Response: $response"
+
+          # Check if the response indicates a package not found error or is not valid JSON
+          if echo "$response" | jq -e '.message == "Package not found." or (.[0].name // empty | length == 0)' >/dev/null 2>&1; then
+            # Set the build number to 69 if no package is found or response is invalid (since the last released was 4.0.0.68)
+            new_build_id=18
```

Comment:
the comment does not match the code 😉

Reply:
I updated it. After the first package is built, we can probably delete this.